packages/anon-aadhaar-pcd/src/types.ts

```diff
@@ -1,4 +1,8 @@
-import { NumberArgument, StringArrayArgument } from '@pcd/pcd-types'
+import {
+  NumberArgument,
+  StringArgument,
```

Comment:
I think we can remove this import if not used. core package will be updated in another PR.

Reply:
Yes, I will remove it another PR I am working on now related to refactor/cleanup

Reply:
Which one? Because I already have some work to push, waiting for develop to be ready.
What is the scope of the refactor/cleanup?